File: Go/factors.go
package main

import (
	"fmt"
	"time"
)

func main() {
	start := time.Now()

	num := 2000000000

	for i := 1; i <= num; i++ {
		if num % i == 0 {
			fmt.Printf("%d is a factor of %v\n", i, num)
		}
	}

	fmt.Printf("Finished in: %v", time.Since(start))
}

File: Rust/src/factors.rs
use std::time::Instant;

fn main() {
    let start = Instant::now();

    const NUM: i64 = 2000000000;
    for i in 1..NUM {
        if NUM % i == 0 {
            println!("{} is a factor of {}", i, NUM);
        }
    }
    let duration = start.elapsed();
    println!("Finished in: {:?}", duration);
}
